.github/workflows/build-test.yml

```diff
@@ -1,6 +1,12 @@
 name: Build, Test
-
+run-name: CI check
 on:
+  pull_request_review:
+    types:
+      - submitted
```

Comment:
<img width="856" alt="image" src="https://github.com/user-attachments/assets/28b6a308-01a7-4e2c-8d57-87855cdb37c7">

This is creating a lot of duplicate runs

My suggestion is to split the CCTP e2e to a separate file and only run that flow on `pull_request_review`